tests/test_routes/test_watch_routes.py

```
@@ -0,0 +1,192 @@
+            mock_llm.return_value = mock_llm_instance
+
+            with patch(
+                "services.watch_service.WatchService.generate_next_message"
+            ) as mock_gen:
+                mock_gen.return_value = "返答です"
+
+                response = csrf_client.post("/api/watch/next", json={})
+
+                assert response.status_code == 200
+                data = response.get_json()
+                # 花子（スピーカーB）の返答
```

Ruff (0.14.6)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)